.github/workflows/gradle-publish-dev.yml

```diff
@@ -44,3 +44,35 @@ jobs:
           tag: ${{ steps.read_property.outputs.value }}-dev
           allowUpdates: true
           prerelease: true
+
+  publish-modrinth:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Downloading artifact
+        uses: actions/checkout@v5
+      - uses: thecodemonkey/action-get-gradle-version@master
+        id: version
+        with:
+          file: "build.gradle.kts"
+      - uses: actions/download-artifact@v5.0.0
+        with:
+          name: "SBA"
+          path: "./"
+
+      - if: github.event.release.prerelease == true
```

Comment:
This should be removed btw. On Iridium we run this on a github release, where as his is configured to run when pushing to a branch